force-app/main/default/objects/Case/validationRules/EDRD_Validate_Case_Owner.validationRule-meta.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>EDRD_Validate_Case_Owner</fullName>
+    <active>true</active>
```

Comment:
@deepakmulamalla Every validation rule should have a brief write up of why and what does the validation rule does for the record. Although a developer or admin would be able to understand by reviewing the formula, we want to make sure that if any business partner looks at the validation rule, they should be able to understand why this validation rule was added in the first place.

Reply:
@cgijeffolsen done.